File: app/src/main/java/ru/fylmr/diploma/crypto/primitives/curve25519/ge_precomp_0.java
package ru.fylmr.diploma.crypto.primitives.curve25519;

public class ge_precomp_0 {

//CONVERT #include "ge.h"

    public static void ge_precomp_0(ge_precomp h) {
        new fe_1(h.yplusx);
        new fe_1(h.yminusx);
        new fe_0(h.xy2d);
    }


}

File: app/src/main/java/ru/fylmr/diploma/crypto/primitives/curve25519/ge_precomp_0.kt
package ru.fylmr.diploma.crypto.primitives.curve25519

fun gePrecomp0(h: ge_precomp) {
    //CONVERT #include "ge.h"
    feOne(h.yplusx)
    feOne(h.yminusx)
    feZero(h.xy2d)
}
